app/main/views/briefs.py

```diff
@@ -204,6 +206,113 @@ def update_brief_status(brief_id, action):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/award', methods=['POST'])
+def award_pending_brief_response(brief_id):
+    json_payload = get_json_from_request()
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if not brief.status == 'closed':
+        abort(400, "Brief is not closed")
+
+    brief_responses = BriefResponse.query.filter(
+        BriefResponse.brief_id == brief.id,
+        BriefResponse.status != 'draft'
+    )
+    # Check that BriefResponse in POST data is associated with this brief
+    if not json_payload['brief_response_id'] in [b.id for b in brief_responses]:
```

Comment:
Actually we could define `brief_response` up here like:
```
brief_reponse = brief_responses.get(BriefResponse.id==json_payload['brief_response_id'])
if not brief_response:
    # BAIL
```
Then we don't have to define it on line 251